### DISCUSSION TITLE
Scan not work

### DISCUSSION THREAD

**Discussion by Tul1911** [stashapp/stash#5425] [Q&A] [answered]
**Describe the bug**
The Scan activity not "scan" new files

**Expected behavior**
after a scan a new scene must be added to library (as Usual)

**Stash Version: (0.27.2):**

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser [Chrome, Edge, Opera]

**Comment by feederbox826** [comment]
what do the logs say

**Comment by BaIanced** [comment]
> what do the logs say

Mine is doing the same thing after updating to latest stash version. I created a new folder to test 3 videos (confirmed to play in vlc/windows media player).

Log shows: 
```
2024-10-24 21:08:25
Info    
Scan finished (82.9274ms)
2024-10-24 21:08:25
Error   
error processing "Y:Completed\\TEST_SCAN\\PMV Haven - Cock Hero - BBCLove.mp4": running ffprobe on "Y:Completed\\TEST_SCAN\\PMV Haven - Cock Hero - BBCLove.mp4": FFProbe encountered an error with <Y:Completed\TEST_SCAN\PMV Haven - Cock Hero - BBCLove.mp4>.
Error JSON:

Error: exit status 1
2024-10-24 21:08:25
Error   
error processing "Y:Completed\\TEST_SCAN\\PMV - Ahe TikTok Splitscreen - Compilation 1 hour.mp4": running ffprobe on "Y:Completed\\TEST_SCAN\\PMV - Ahe TikTok Splitscreen - Compilation 1 hour.mp4": FFProbe encountered an error with <Y:Completed\TEST_SCAN\PMV - Ahe TikTok Splitscreen - Compilation 1 hour.mp4>.
Error JSON:

Error: exit status 1
2024-10-24 21:08:25
Error   
error processing "Y:Completed\\TEST_SCAN\\Massive Cumshot Compilations.ts": running ffprobe on "Y:Completed\\TEST_SCAN\\Massive Cumshot Compilations.ts": FFProbe encountered an error with <Y:Completed\TEST_SCAN\Massive Cumshot Compilations.ts>.
Error JSON:

Error: exit status 1
2024-10-24 21:08:25
Info    
Calculating fingerprints for Y:Completed\TEST_SCAN\Massive Cumshot Compilations.ts ...
2024-10-24 21:08:25
Info    
Calculating fingerprints for Y:Completed\TEST_SCAN\PMV - Ahe TikTok Splitscreen - Compilation 1 hour.mp4 ...
2024-10-24 21:08:25
Info    
Calculating fingerprints for Y:Completed\TEST_SCAN\PMV Haven - Cock Hero - BBCLove.mp4 ...
2024-10-24 21:08:25
Info    
Finished adding files to queue. 3 files queued
2024-10-24 21:08:25
Info    
Y:Completed\TEST_SCAN doesn't exist. Creating new folder entry...
2024-10-24 21:08:25
Info    
scanning 1 paths
```

**Comment by feederbox826** [comment]
can you post the error JSON?

**Comment by BaIanced** [comment]
> can you post the error JSON?

I don't know what JSON file it's referring to, but what I posted above is verbose trace log

**Comment by HereIam80** [comment]
I see a missing backslash in your path.

**Comment by BaIanced** [comment]
After a little digging:
I figured out that the .stash directory was set to read-only somehow...
After deselecting that option in the folder properties, the next scan took nearly an hour to complete, but it successfully added the new clips. 
Scan is back to normal (not taking an hour), so OP maybe check the folder is writeable.

**Comment by BaIanced** [comment]
> I see a missing backslash in your path.

for whatever reason stash allows directories to be input with or without a backslash

**Comment by snohotest** [comment]
Also had this issue after upgrading from 0.25.1 -> 0.27.2, but am on Linux and wasn't having any permissions issue. I got this working again by just deleting `ffprobe` inside of my `.stash` folder.

**Comment by MixingMA** [comment]
Having the same issue. Path isn't read only and ffprobe can access the files directly, but getting the exit status 1 error with stashapp. Deleting ffprobe from my .stash folder just gives me a new error that ffprobe isn't found.

**Reply by DogmaDragon** [reply, marked as answer]
Delete old ffmpeg/ffprobe and download the latest version.

Settings > System > Application Paths > Download FFmpeg.

**Reply by MixingMA** [reply]
That did it, thank you!
